cockroachdb-in-comparison.md

```diff
@@ -2,3 +2,19 @@
 title: CockroachDB in Comparison
 toc: false
 ---
+
+| | CockroachDB | MySQL | PostgreSQL | Cassandra | HBase | MongoDB | Riak 
+--|--|--|--|--|--|--|--
+Automated Scaling | Yes | No | No | Yes | Yes  | Yes | Yes 
+Automated Failover & Repair | Yes | No | No | Yes | Yes | Yes | Yes 
```

Comment:
It isn't currently possible through SQL, though that would be relatively easy to change. For now we can go ahead and say "No" for that feature I suppose.